SPA/README.md

```diff
@@ -1,25 +1,28 @@
-# Auth0 jQuery Single Page App Account Linking Sample
+# Auth0 SPA (Vanilla JS) Account Linking Sample
 
-This Single Page App serves as an example of the [User Initiated Account Linking Scenario](https://auth0.com/docs/link-accounts/user-initiated), where the user can click on a **link account** button and login with other identity to link to. 
+This Single Page App serves as an example of the [User Initiated Account Linking Scenario](https://auth0.com/docs/link-accounts/user-initiated), where the user can click on a **Link Account** button and login with other identity to link to.
+
+**For security reasons, the user performing the account linking and the account being linked must both email_verified set to true. The sample will enforce this constrain during account linking.**
 
 ## Key Features
 
-* Login with any available connection using [Lock](https://github.com/auth0/lock)
-* Login with any passwordless connections, using [Lock Passwordless](https://github.com/auth0/lock-passwordless)
-* Display of current profile and already linked accounts
-* Option to unlink an account
-* Option to link another account
-* Option to link another account with a specific connection (i.e. facebook, twitter, etc)
-* Option to link to a passwordless account
+- Login with any available connection using [Universal Login](https://auth0.com/docs/universal-login)
+- Display of current profile and already linked accounts
+- Option to unlink an account
+- Option to link another account
 
-![](spa-user-settings.png)
+![](pngs/spa-account-linking.png)
 
 ## Install Locally
 
-1. Create an auth0-variables.js file with your Auth0 credentials. You can use auth0-variables.sample.js as a template. You can get the clientId and domain from the [Auth0 Dashboard](https://manage.auth0.com).
-2. In your App's configuration on the [Auth0 Dashboard](https://manage.auth0.com), add `http://localhost:3000` to the list of **Allowed Callback URLs**. 
-3. Initialize a web server in the samples folder. You can do it for instance with `serve`:
-	* Install node
-	* run `npm install -g serve`
-	* run `serve` in the project's folder to start a server
-4. Go to the [index page](http://localhost:3000) and start playing with the app! 
\ No newline at end of file
+1. Make a copy of `auth_config.json.example` and rename it to `auth_config.json`. Then open it in a text editor and supply the client_id and domain values for your application. You can get the client_id and domain from the [Auth0 Dashboard](https://manage.auth0.com).
+2. In your App's configuration on the [Auth0 Dashboard](https://manage.auth0.com), add `http://localhost:3000` to the list of **Allowed Callback URLs**, **Allowed Logout URLs** and **Allowed Web Origins**.
+
+3. From the SPA folder, run:
+
+```
+- npm install
+- npm start
+```
+
+4. Go to the [index page](http://localhost:3000) and start playing with the app!
```

Comment:
Should we clarify this sample will not work with 3rd party cookies disabled? I have a browser configured to block cookies and I had to turn that off to make it work.

Reply:
https://github.com/auth0-samples/auth0-link-accounts-sample/pull/13/commits/836966f9b200c69dcb7d41063f02421df00eb187